newbank/server/Offer.java

```diff
@@ -62,4 +62,33 @@ public int getBorrowingLengthInMonth() {
   public void borrow(BigDecimal amountToBorrow) {
     this.amount = this.amount.subtract(amountToBorrow);
   }
+  
+  private static int getNextOfferNumber() {
+    return MicroLoanMarketPlace.getNextOfferNumber();
+  }
+  
+  @Override
+  public String toString() {
+    return String.format(
+        "%s: %d"
+            + System.lineSeparator()
+            + "%s: %s"
+            + System.lineSeparator()
+            + "%s: %s"
+            + System.lineSeparator()
+            + "%s: %d"
+            + System.lineSeparator(),
+        "Offer Number", this.offerNumber,
+        "Interest Rate", new DecimalFormat("0.00").format(this.interestRate),
```

Comment:
Hi Simon could the format be changed here to write a percentage rather than a decimal to make it easier to understand for customers?

Reply:
Hi Heather all done.  Noticed in testing that the OfferCommand rounds to the nearest %.